nodejs/eks/examples/migrate-nodegroups/scripts/load-testing.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
This isn't used currently - right?  Does it makes sense to put this in an example and/or blog post instead of in this repo if unused?

Reply:
Removed